gov-output/mainnet/add-collateral-34858172.json

```diff
@@ -0,0 +1,70 @@
+{
+  "CollateralAuctionHouseParams": {
+    "maxDiscount": "900000000000000000",
+    "minimumBid": "100000000000000000000",
+    "minDiscount": "1000000000000000000",
+    "perSecondDiscountUpdateRate": "999985752120000000000000000"
+  },
+  "GlobalSettlement_Address": "0x1c6B7ab018be82ed6b5c63aE82D9f07bb7B231A2",
+  "LiquidationEngineCollateralParams": {
+    "newCAHChild": "0x293b702362918e990Dd5f48a80fB139FeAd64263",
+    "liquidationPenalty": "1050000000000000000",
+    "liquidationQuantity": "50000000000000000000000000000000000000000000000000"
```

Comment:
the liquidation quantity for RETH on mainnet is:
100000000000000000000000000000000000000000000000000
here it's shows:
50000000000000000000000000000000000000000000000000
(just noting the difference, I know it 50k od vs 100k od)
liquidation penalty is the same